tests/app/main/test_briefs.py

```diff
@@ -0,0 +1,134 @@
+import mock
+from dmapiclient import HTTPError
+from dmutils.email import MandrillException
+
+from ..helpers import BaseApplicationTest, FakeMail
+
+
+@mock.patch('app.main.views.briefs.data_api_client', autospec=True)
+class TestBriefClarificationQuestions(BaseApplicationTest):
+    def test_clarification_question_form_requires_login(self, data_api_client):
+        res = self.client.get('/suppliers/opportunities/1/ask-a-question')
+        assert res.status_code == 302
+        assert '/login' in res.headers['Location']
+
+    def test_clarification_question_form(self, data_api_client):
+        self.login()
+        data_api_client.get_brief.return_value = {'briefs': {'status': 'live'}}
+
+        res = self.client.get('/suppliers/opportunities/1/ask-a-question')
+        assert res.status_code == 200
+
+    def test_clarification_question_form_requires_existing_brief_id(self, data_api_client):
+        self.login()
+        data_api_client.get_brief.side_effect = HTTPError(mock.Mock(status_code=404))
+
+        res = self.client.get('/suppliers/opportunities/1/ask-a-question')
+        assert res.status_code == 404
+
+    def test_clarification_question_form_requires_live_brief(self, data_api_client):
+        self.login()
+        data_api_client.get_brief.return_value = {'briefs': {'status': 'expired'}}
+
+        res = self.client.get('/suppliers/opportunities/1/ask-a-question')
+        assert res.status_code == 404
+
+
+@mock.patch('app.main.views.briefs.data_api_client', autospec=True)
+class TestSubmitClarificationQuestions(BaseApplicationTest):
+    def test_submit_clarification_question_requires_login(self, data_api_client):
+        res = self.client.post('/suppliers/opportunities/1/ask-a-question')
+        assert res.status_code == 302
+        assert '/login' in res.headers['Location']
+
+    @mock.patch('app.main.helpers.briefs.send_email')
+    def test_submit_clarification_question(self, send_email, data_api_client):
+        self.login()
+        data_api_client.get_brief.return_value = {'briefs': {
```

Comment:
Kev was using [some generated brief from dmutils](https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/pc_supplier_reply_responses/tests/app/main/test_briefs.py#L46) for tests.  Not sure if it's easy to change  / worth changing your existing stuff (but it might be nice to be consistent).